docs/package.json

```diff
@@ -4,7 +4,7 @@
     "private": true,
     "scripts": {
         "build": "next build",
-        "dev": "next dev",
+        "dev": "next dev --turbo",
```

Comment:
This is a new version of Next.js that recommends Turbopack as the dev server by default.